crates/bevy_ui/src/entity.rs

```diff
@@ -39,6 +42,38 @@ pub struct NodeBundle {
     pub computed_visibility: ComputedVisibility,
 }
 
+impl NodeBundle {
+    pub fn new(style: Style, color: impl Into<UiColor>) -> Self {
```

Comment:
While I like this method a lot, I think using the name `new` here is confusing. I'm use to `new` being the generic constructor, but this constructor is clearly meant as a "simpler, common-case constructor". Maybe a name like `basic` would make more sense?

Reply:
Yeah, I wasn't sure about that one either. It's pretty common in languages with variadics to just have a bunch of `new` with different overloads, but in this case the rust way would probably be something like `NodeBundle::from_style_and_color()` but that's way too long for something that common. I'm not a big fan of `basic` but right now I don't have any better idea. Maybe `NodeBundle::styled()`.

Reply:
I like `NodeBundle::styled()`

Reply:
Me too!